run_model-dep-full_limits_per_channel.sh

```diff
@@ -113,20 +105,28 @@ if [[ $MODE == "initial" ]]; then
     ############
     # morphing
     ############
-    # morph_parallel.py --output ${defaultdir}/datacards \
-    #     --analysis ${analysis} \
-    #     --eras $ERA \
-    #     --category_list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/by_channel/sm_neuralnet_categories_$CHANNEL.txt \
-    #     --variable nnscore \
-    #     --sm \
-    #     --parallel 10 2>&1 | tee -a ${defaultdir}/logs/morph_sm_log.txt
+    morph_parallel.py --output ${defaultdir}/datacards \
+        --analysis ${analysis} \
+        --sub-analysis ${sub_analysis} \
+        --categorization ${categorization} \
+        --sm-like-hists ${sm_like_hists} \
+        --sm-gg-fractions ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/data/higgs_pt_reweighting_fullRun2_v2.root \
+        --eras 2016,2017,2018 \
+        --category_list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/by_channel/sm_neuralnet_categories_$CHANNEL.txt \
```

Comment:
Shouldn't this be with `-` as well?
```suggestion
        --category-list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/by_channel/sm_neuralnet_categories_$CHANNEL.txt \
```

Reply:
Good spot!